### PR TITLE
feat: update readme and docs folder.

### PR DESCRIPTION
**Pull request by vaultec81**
This PR includes:
- Documentation & readme updates
- Codecov CI integration

### CODE REVIEW

**Review by kanej — APPROVED** [review]
Looks good

**Comment by kanej** [comment]
Not sure what is happening with the codecov, but we are good to merge once that is resolved.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,70 @@
...
+
+<p align="center">
+  <a href="https://vimeo.com/489472356">
+    <img src="docs/media/ms_1.png">
+  </a>
+<p>
+
+# Table of Contents 📚
 <!-- toc -->
+* [Features](#features)
```
Comment (AquiGorka) [inline README.md:34]: These links are not working, it must have something to do with the section titles having space and an emoji; when I click directly on the title anchor elements the links show an extra `-` in the url bar which the menu anchors do not have. Minor details but let's fix it.

Comment (vaultec81) [reply README.md]: Fixed: Need `-(emoji)`

**Pull request merged**